Documentation/gettingstarted/aws.rst

```diff
@@ -0,0 +1,303 @@
+
+
+Validate that derived policy is in place
+----------------------------------------
+
+Every time that a new policy with ToGroups rules is added a new derivative
+policy will be created, the derivative policy will contain the list of allowed
+IPs in the ToCIDRSet. This new policy will be updated every five minutes to get
+AWS instances changes.
+
+.. parsed-literal::
+
+    $ kubectl get cnp
+    NAME                                                             AGE
+    to-groups-sample                                                 11s
+    to-groups-sample-togroups-044ba7d1-f491-11e8-ad2e-080027d2d952   10s
+
+Where the derivative policy has the following entries on the ToCIDR:
+
+.. parsed-literal::
+
+   $ kubectl get cnp to-groups-sample-togroups-044ba7d1-f491-11e8-ad2e-080027d2d952
+
+
+.. code-block:: yaml
+
+    apiVersion: cilium.io/v2
+    kind: CiliumNetworkPolicy
+    metadata:
+      creationTimestamp: 2018-11-30T11:13:52Z
+      generation: 1
+      labels:
+        io.cilium.network.policy.kind: derivative
+        io.cilium.network.policy.parent.uuid: 044ba7d1-f491-11e8-ad2e-080027d2d952
+      name: to-groups-sample-togroups-044ba7d1-f491-11e8-ad2e-080027d2d952
+      namespace: default
+      ownerReferences:
+      - apiVersion: cilium.io/v2
+        blockOwnerDeletion: true
+        kind: CiliumNetworkPolicy
+        name: to-groups-sample
+        uid: 044ba7d1-f491-11e8-ad2e-080027d2d952
+      resourceVersion: "34853"
+      selfLink: /apis/cilium.io/v2/namespaces/default/ciliumnetworkpolicies/to-groups-sample-togroups-044ba7d1-f491-11e8-ad2e-080027d2d952
+      uid: 04b289ba-f491-11e8-ad2e-080027d2d952
+    specs:
+    - egress:
+      - toCIDRSet:
+        - cidr: 34.254.113.42/32
+        - cidr: 172.31.44.160/32
+        toPorts:
+        - ports:
+          - port: "80"
+            protocol: TCP
+      endpointSelector:
+        matchLabels:
+          any:class: xwing
+          any:org: alliance
+          k8s:io.kubernetes.pod.namespace: default
+      labels:
+      - key: io.cilium.k8s.policy.name
+        source: k8s
+        value: to-groups-sample
+      - key: io.cilium.k8s.policy.uid
+        source: k8s
+        value: 044ba7d1-f491-11e8-ad2e-080027d2d952
+      - key: io.cilium.k8s.policy.namespace
+        source: k8s
+        value: default
+      - key: io.cilium.k8s.policy.derived-from
+        source: k8s
+        value: CiliumNetworkPolicy
+    status:
+      nodes:
+        k8s1:
+          enforcing: true
+          lastUpdated: 2018-11-30T11:28:03.907678888Z
+          localPolicyRevision: 28
+          ok: true
+
+The important information that the derivative rule should have is:
+
+- *metadata.OwnerReferences*: that contains the information about the parent
+  policy.
+
+- *specs.Egress.ToCIDRSet*: that is the list of the instances private and public
+  IPs for the parent to groups filter.
+
+- *status*: that is the policy enforce status and when is updated.
```

Comment:
whether or not the policy is enforced yet, and when the policy was last updated.